src/OrchardCore.Modules/OrchardCore.Contents/Indexing/AspectsContentIndexHandler.cs

```diff
@@ -36,6 +38,16 @@ public async Task BuildIndexAsync(BuildIndexContext context)
                 context.ContentItem.DisplayText,
                 DocumentIndexOptions.Store);
 
+            context.DocumentIndex.Set(
```

Comment:
Is it only for ordering?

Reply:
Yes, but it should be removed in favor of the Tokenized field implementation. The Liquid filters should then allow to do ReplaceDiacritics.